statics/js/components/inject-form.js

```diff
@@ -33,6 +34,16 @@ Vue.component('inject-form', {
         <label for="inject-count">ICMP Identifier</label>\
         <input id="inject-id" type="number" class="form-control input-sm" v-model="id" min="0" />\
       </div>\
+      <div v-if="true">\
+        <div class="form-group">\
+          <label for="src-port">Src Port</label>\
```

Comment:
hi @safchain , i need help on this when i try to hide the div by using the value of select's option i am getting "Unexpected identifier". below is the code at line number 37,
<div v-if="type === 'tcp'">
need some hint on this